test/service/test_repository_projects.py

```diff
@@ -28,63 +29,34 @@ def get_project_detail_properties():
         """Get a list of properties that are only returned in the /repository/projects/{id} response"""
         return ['contributors', 'projectDescription', 'publications']
 
-    def test_list_response(self):
+    def test_project_response(self):
         """
         Make call to endpoint that returns multiple projects
-        A list of hits should be returned
-        Certain fields should not be in the project object
+        Verify response and verify a single hit matches with
+        response given for call to endpoint with project uuid specified
         """
-        url = self.base_url + '/repository/projects'
-        response = requests.get(url)
-        response.raise_for_status()
-        response_json = response.json()
 
-        def assert_file_type_summaries(hit):
-            self.assertEqual(len(hit['fileTypeSummaries']), 1)
-            self.assertIn('fileType', hit['fileTypeSummaries'][0])
-            self.assertGreater(hit['fileTypeSummaries'][0]['count'], 0)
-            self.assertGreater(hit['fileTypeSummaries'][0]['totalSize'], 0)
```

Comment:
This method is duplicated in the [samples repository endpoint test](https://github.com/DataBiosphere/azul/blob/acbcbc7134cad74c2c464e20825602e6b8cc2799/test/service/test_repository_specimen.py#L30). Instead of removing it, these duplicates should be consolidated. I'll leave it up to @hannes-ucsc to decide whether that change belongs in this PR or not, but in the meantime I don't think the method should be removed.

Reply:
Ok, I'll restore the method for now

Reply:
Good call. I feel like the review improved the quality of the PR and made my life easier.